package.json

```diff
@@ -14,6 +14,7 @@
     "clean": "wsrun -p $(yarn --silent run pkgparse) -r --fast-exit --parallel --exclude-missing clean",
     "clean_modules": "rm -rf node_modules && find ./packages -type d -maxdepth 2 -name \"node_modules\" -exec rm -r {} +",
     "test": "wsrun -p $(yarn --silent run pkgparse) --fast-exit --no-prefix --exclude-missing --timeout 20000 test",
+    "test:local": "NODE_ENV=local wsrun -p @eth-optimism*/integration-tests --fast-exit --no-prefix --exclude-missing --timeout 20000 test && NODE_ENV=local wsrun -p @eth-optimism*/tx-ingestion --fast-exit --no-prefix --exclude-missing --timeout 20000 test",
```

Comment:
This should follow the same `PKGS` convention with `$(yarn --silent run pkgparse)` instead of hardcoding it with a specific test suite